libs/helperHashEncode.js

```diff
@@ -9,6 +9,7 @@ import Hashids from 'hashids';
  * @returns {string} Hashed personal number
  */
 function encode(number, hashSalt, hashSaltLength) {
+  typeof number === 'number' || (number = parseInt(number, 10));
```

Comment:
Unclear expression, didn't catch that there is both an evaluation and conditional assignment...please refactor to more clear code 👌.

```javascript
if(typeof number !== 'number') {
  number = parseInt(number, 10);
}
```

But is this check even necessary? I'm not a fan of reassigning arguments either...